sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.aest.controller.getScholasticStandingSummary.e2e-spec.ts

```diff
@@ -0,0 +1,95 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeStudentScholasticStanding,
+  saveFakeApplication,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import * as request from "supertest";
+import { StudentScholasticStanding } from "@sims/sims-db";
+import { saveFakeSFASIndividual } from "@sims/test-utils/factories/sfas-individuals";
+import { Repository } from "typeorm";
+
+describe("StudentScholasticStandingsAESTController(e2e)-getScholasticStandingSummary.", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let scholasticStandingRepo: Repository<StudentScholasticStanding>;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    scholasticStandingRepo = db.dataSource.getRepository(
```

Comment:
You don't need do this. Just do:
`db.studentScholasticStanding.save()`

Reply:
The `saveFake*` methods are intended to be used when multiple persistent operations must be executed, for instance, a user must be created to allow a student to be created or something similar.
Do we need this method right now?